modules/sdk-coin-avaxp/src/avaxp.ts

```diff
@@ -77,17 +79,47 @@ export class AvaxP extends BaseCoin {
     }
     switch (explainedTx.type) {
       case TransactionType.AddDelegator:
+        break;
       case TransactionType.AddValidator:
         if (!params.txParams.recipients || params.txParams.recipients.length === 0) {
-          const filteredRecipients = [{ address: stakingOptions.nodeID, amount: stakingOptions.amount }];
+          const filteredRecipients = [{ address: stakingOptions?.nodeID, amount: stakingOptions?.amount }];
           const filteredOutputs = explainedTx.outputs.map((output) => _.pick(output, ['address', 'amount']));
 
           if (!_.isEqual(filteredOutputs, filteredRecipients)) {
             throw new Error('Tx outputs does not match with expected txParams');
           }
-          if (stakingOptions.amount !== explainedTx.outputAmount) {
+          if (stakingOptions?.amount !== explainedTx.outputAmount) {
+            throw new Error('Tx total amount does not match with expected total amount field');
+          }
+        } else {
+          throw new Error('Stake Tx does not required recipients');
+        }
+        break;
+      case TransactionType.Export:
+        if (params.txParams.recipients) {
+          if (params.txParams.recipients.length > 1) {
+            throw new Error('Export Tx requires one recipient');
+          }
+          let totalAmount = new BigNumber(0);
+          for (const recipients of params.txParams.recipients) {
+            totalAmount = totalAmount.plus(recipients.amount);
+          }
+          if (!totalAmount.isEqualTo(explainedTx.outputAmount)) {
```

Comment:
@CapnMigraine Are we  going to estimate import C fee to be add it in the export amount?